app/scripts/views/mixins/settings-panel-mixin.js

```diff
@@ -29,6 +42,15 @@ define(function (require, exports, module) {
 
     openPanel: function () {
       this.$('.settings-unit').addClass('open');
+      var input = this.$('.open input:first-of-type');
+      if (input.length > 0) {
+        input[0].focus();
```

Comment:
Calling `focus` directly can have bad consequences in IE if the element is not visible. Can you call `this.focus(input)` which [handles those problems for you](https://github.com/mozilla/fxa-content-server/blob/7c5aee43ab38b99e840d52d4478957c4def31e35/app/scripts/views/base.js#L669:L686)?